Firestore/Example/FuzzTests/FSTFuzzTestsPrincipal.mm

```diff
@@ -18,16 +18,23 @@
 
 #include "LibFuzzer/FuzzerDefs.h"
 
+#include "Firestore/core/src/firebase/firestore/model/database_id.h"
 #include "Firestore/core/src/firebase/firestore/remote/serializer.h"
 
+using firebase::firestore::model::DatabaseId;
 using firebase::firestore::remote::Serializer;
 
 namespace {
 
+// Global static objects. Initialized before fuzzing starts.
+static Serializer *serializer;
+
```

Comment:
Just wanted to point out that global primitives are, unlike local variables, initialized to a valid value (some version of zero), so this is a null pointer, which is great.